suppressible-error-prone/src/main/java/com/palantir/suppressibleerrorprone/VisitorStateModifications.java

```diff
@@ -135,8 +138,50 @@ public static Description interceptDescription(VisitorState visitorState, Descri
                 .build();
     }
 
-    private static boolean suppressibleTree(Tree tree) {
-        return modifiersTree(tree).isPresent();
+    private static boolean suppressibleTreePath(TreePath treePath) {
+        if (!modifiersTree(treePath.getLeaf()).isPresent()) {
+            return false;
+        }
```

Comment:
At this point, I'm inclined to just make this a check for whether it's an instance of ClassTree, MethodTree or VariableTree. Being suppressible isn't actually tied to having a ModifiersTree it would seem